src/UE4SSProgram.cpp

```diff
@@ -916,27 +930,11 @@ namespace RC
         for (auto& mod : m_mods)
         {
             // Remove any actions, or we'll get an internal error as the lua ref won't be valid
-            mod->clear_delayed_actions();
```

Comment:
Was this intended ?
I'm not seeing any calls to `clear_delayed_actions` anywhere else and we need to clear the delayed actions when uninstalling.
I'm guessing this was meant to be moved into the `uninstall` function in the `LuaMod` class.